apps/staking/providers/feature-flag-provider.tsx

```diff
@@ -1,142 +0,0 @@
-    FEATURE_FLAG.MOCK_OPEN_NODES,
-    FEATURE_FLAG.MOCK_NO_OPEN_NODES,
-    FEATURE_FLAG.MOCK_PENDING_NODES_ONE,
-    FEATURE_FLAG.MOCK_PENDING_NODES_TWO,
-    FEATURE_FLAG.MOCK_PENDING_NODES_THREE,
-    FEATURE_FLAG.MOCK_PENDING_NODES_MANY,
-    FEATURE_FLAG.MOCK_NO_PENDING_NODES,
-  ],
-  register: [FEATURE_FLAG.MOCK_REGISTRATION],
-};
-
-export const globalFeatureFlags = [FEATURE_FLAG.CLEAR_ACCEPT_TOS];
-
-type FeatureFlags = Record<FEATURE_FLAG, boolean>;
-
-type FeatureFlagContext = {
-  featureFlags: FeatureFlags;
-  setFeatureFlags: (featureFlags: FeatureFlags) => void;
-};
-
-const localStorageFeatureFlagKey = 'featureFlags';
-
-function saveFeatureFlagsToStorage(featureFlags: FeatureFlags) {
-  localStorage.setItem(localStorageFeatureFlagKey, JSON.stringify(featureFlags));
-
-  // @ts-expect-error - Expose feature flags to window for debugging
-  window.featureFlags = featureFlags;
-}
-
-function loadFeatureFlagsFromStorage() {
-  const storedFeatureFlags = localStorage.getItem(localStorageFeatureFlagKey);
-
-  if (!storedFeatureFlags) {
-    return {};
-  }
-
-  return JSON.parse(storedFeatureFlags);
-}
-
-function bootstrapFeatureFlags() {
-  if (typeof window === 'undefined') {
-    return {};
-  }
-  const featureFlags = loadFeatureFlagsFromStorage();
-  const searchParams = useSearchParams();
-
-  const searchParamEntries = searchParams.entries();
-
-  for (const [key, value] of searchParamEntries) {
-    if (validFeatureFlags.includes(key as FEATURE_FLAG)) {
-      featureFlags[key as FEATURE_FLAG] = value === 'true';
-    }
-  }
-
-  saveFeatureFlagsToStorage(featureFlags);
-
-  return featureFlags;
-}
-
-const Context = createContext<FeatureFlagContext | undefined>(undefined);
-
-export default function FeatureFlagProvider({ children }: { children: React.ReactNode }) {
-  const [featureFlags, setFeatureFlags] = useState(bootstrapFeatureFlags());
-
-  return <Context.Provider value={{ featureFlags, setFeatureFlags }}>{children}</Context.Provider>;
-}
-
-export const useFeatureFlags = () => {
-  const context = useContext(Context);
-
-  if (context === undefined) {
-    throw new Error('useFeatureFlags must be used inside FeatureFlagProvider');
-  }
-
-  return context.featureFlags;
-};
-
-export const useFeatureFlag = (flag: FEATURE_FLAG) => {
-  const context = useContext(Context);
-
-  if (context === undefined) {
-    throw new Error('useFeatureFlag must be used inside FeatureFlagProvider');
-  }
-
-  return context.featureFlags[flag] ?? false;
-};
-
-export const useSetFeatureFlag = () => {
-  const context = useContext(Context);
-
-  if (context === undefined) {
-    throw new Error('setFeatureFlag must be used inside FeatureFlagProvider');
-  }
-
-  const setFeatureFlag = (flag: FEATURE_FLAG, value: boolean) => {
-    const newFeatureFlags = { ...context.featureFlags, [flag]: value };
-
-    context.setFeatureFlags(newFeatureFlags);
-
-    saveFeatureFlagsToStorage(newFeatureFlags);
-  };
-
-  return { setFeatureFlag };
```

Comment:
I get a build error when I do `pn build` from file `apps/staking/providers/global-provider.tsx` 

```
Module not found: Can't resolve '@/providers/feature-flag-provider'
```

Reply:
This is fixed in the next PR, my bad, the file is deleted